src/components/Footer.astro

```diff
@@ -42,9 +42,9 @@ import Socials from "./Socials.astro";
     {
       !!DATA.previousEditions.length && (
         <div class="flex flex-col gap-6 w-fit mx-auto">
-          <h5 class="text-center lg:text-left font-semibold text-lg">
```

Comment:
this is not supposed to be an h2, due to footer section having a lower level of impact than main content sections 
![image](https://github.com/user-attachments/assets/e5120a32-658c-460e-87f0-f5a6c251cf3e)
![image](https://github.com/user-attachments/assets/cbcc303d-7da8-4738-93cb-e1fcc20833a0)

Reply:
By keeping it as '<h5>', the heading is semantically like a part of the latest section:
![image](https://github.com/user-attachments/assets/ff990103-44b9-4792-8fd2-9b825231d36c)